Gemfile

```diff
@@ -4,7 +4,7 @@ ruby '2.4.1'
 
 gem 'api-pagination'
 gem 'apipie-rails'
-gem 'audited-activerecord', '~> 4.0'
+gem 'audited', '~> 4.5'
```

Comment:
Looks like `audited` is the new version of `audited-activerecord`, and the one that works with Rails 5.0